docs/src/formatBidsAndAsks.md.ejs

```diff
@@ -44,15 +44,15 @@ const address = 'r9cZA1mLK5R5Am25ArfXFmqgNwjZgnfk59';
 return Promise.all(
   [
     this.api.request('book_offers', {
-      taker_gets: RippleAPI.renameCounterpartyToIssuer(orderbookInfo.base),
-      taker_pays: RippleAPI.renameCounterpartyToIssuer(orderbookInfo.counter),
+      taker_gets: RippleAPI.renameissuerToIssuer(orderbookInfo.base),
+      taker_pays: RippleAPI.renameissuerToIssuer(orderbookInfo.counter),
```

Comment:
This is not a method. And I also don't think you can update docs directly.

Reply:
Got it!